Replicator/Replicator.cc

```diff
@@ -56,6 +57,24 @@ namespace litecore { namespace repl {
     };
 
 
+    std::vector<string> Replicator::CompatibleProtocols() {
```

Comment:
I don't think this needs to be dynamically constructed every time it is created.  I think a static C array of fully constructed names should be fine given how infrequently kWSProtocolName will change.  At least making the construction of this happen only once would be better.